src/components-examples/angular/input/input-form/input-form-example.html

```diff
@@ -0,0 +1,57 @@
+<form class="example-form">
+  <sbb-form-field class="example-full-width">
+    <sbb-label>Company (disabled)</sbb-label>
+    <input sbbInput disabled value="SBB" />
+  </sbb-form-field>
+
+  <table class="example-full-width" cellspacing="0">
```

Comment:
I think using a table to group elements is not the way we should provide an our showcase. css grid?